Dockerfile

```
@@ -2,6 +2,7 @@ ARG BASE_IMAGE
 FROM $BASE_IMAGE
```

This will force usage of the amd64 image even when running the build on an arm64 machine, to prevent accidentally building the wrong architecture:

```suggestion
FROM --platform=linux/amd64 $BASE_IMAGE
```